### PR TITLE
sprint 6 code added

### PR DESCRIPTION
**Pull request by Sachin-Rajpoot-NucleusTeq**

### CODE REVIEW

File: backend/gms/src/main/java/com/gms/demo/controller/MemberController.java [hunk 1]
```diff
@@ -75,19 +62,11 @@ public ResponseEntity<?> login(
           memberLoginDto.getPassword()
         )
     );
-
```
Comment (Sejal-NucleusTeq) [inline backend/gms/src/main/java/com/gms/demo/controller/MemberController.java]: please remove commented code

File: backend/gms/src/main/java/com/gms/demo/controller/MemberController.java [hunk 2]
```diff
@@ -100,7 +79,8 @@ public ResponseEntity<?> login(
    * @param memberDto The MemberDto containing member data.
    * @param email     The email header used for verification.
    * @param password  The password header used for verification.
-   * @return A ResponseEntity containing the created MemberOutDto and an HTTP status code.
+   * @return A ResponseEntity containing the created MemberOutDto 
+   *     and an HTTP status code.
    */
   @CrossOrigin
   @PostMapping("create/nodept")
```
Comment (Sejal-NucleusTeq) [inline backend/gms/src/main/java/com/gms/demo/controller/MemberController.java:86]: what is nodept here?

File: backend/gms/src/main/java/com/gms/demo/controller/TicketController.java [hunk 3]
```diff
@@ -42,32 +42,37 @@ public class TicketController {
...
    */
   @CrossOrigin
   @PostMapping("create/memberId/{memberId}/departmentId/{departmentId}")
-  public ResponseEntity<?> createTicket(
+  public final ResponseEntity<?> createTicket(
       @Valid @RequestBody final TicketDto ticketDto,
       @PathVariable final Integer memberId,
       @PathVariable final Integer departmentId
   ) {
     System.out.println("INSIDE CREATE MEMBER");
```
Comment (Sejal-NucleusTeq) [inline backend/gms/src/main/java/com/gms/demo/controller/TicketController.java]: instead of sysout please use logger

Comment (Sachin-Rajpoot-NucleusTeq) [reply backend/gms/src/main/java/com/gms/demo/controller/TicketController.java]: Yes, I will add loggers today

File: frontend/src/Components/Tables/TicketTable.jsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import "./TicketTable.css";
 import PopUp from "../PopUp/PopUp";
 
 function TicketTable({ isLoggedIn }) {
-  console.log("TTTTTTTTTTTTTTTTTTTTTTTT")
+  console.log("TTTTTTTTTTTTTTTTTTTTTTTT");
```
Comment (Sejal-NucleusTeq) [inline frontend/src/Components/Tables/TicketTable.jsx:7]: give appropriate msgs in console as well.

**Pull request merged**